tests/unit_tests/test_wave_06.py

```diff
@@ -152,7 +154,20 @@ def test_swap_best_by_category_reordered():
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories, and that the items that were swapped are not there
 
-@pytest.mark.skip
+
+    # Assert
+    assert result
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_a in tai.inventory
+    assert item_b in tai.inventory
+    assert item_f in tai.inventory  
+    assert item_c not in tai.inventory  
+    assert item_c in jesse.inventory 
+    assert item_d in jesse.inventory
+    assert item_e in jesse.inventory
+    assert item_f not in jesse.inventory  
```

Comment:
Same idea as the previous test, if we know what jesse and tai's inventories should look like, we can hardcode lists that match what we should see and assert that they are equal. This saves us the trouble of having to check and see if each item is in its respective inventory.